packages/amplify-codegen/src/commands/models.js

```diff
@@ -305,7 +305,7 @@ function loadSchema(apiResourcePath) {
   }
   if (fs.pathExistsSync(schemaDirectory) && fs.lstatSync(schemaDirectory).isDirectory()) {
     // search recursively for graphql schema files inside `schema` directory
-    const schemas = glob.sync([path.join(schemaDirectory, '**/*.graphql')]);
+    const schemas = globby.sync([path.join(schemaDirectory, '**/*.graphql')].map((path) => path.replace(/\\/g, '/')));
```

Comment:
I believe we used node's built-in `path` to enforce the correct path parsing & separator in another context. I can't find the reference, but maybe @dpilch remembers?

Reply:
This is what I used to switch the separators: https://github.com/aws-amplify/amplify-codegen/blob/1ffbaf10745d9694dd0b5f5308a54af2ef6588fb/packages/amplify-codegen/src/utils/getRelativeTypesPath.js#L6-L9